.ruby-version

```diff
@@ -1 +1 @@
-2.5.9
+3.0.0
```

Comment:
I would suggest to use same version than the one used in the Dockerfile.
```bash
3.1.2
```

Reply:
Yes you are completely right thank you.